tailcfg/tailcfg.go

```diff
@@ -1624,6 +1624,10 @@ type SSHAction struct {
 	// response, it should be re-fetched as long as the SSH
 	// session is open.
 	HoldAndDelegate string `json:"holdAndDelegate,omitempty"`
+
+	// LocalPortForwarding, if true, allows accepted connections
+	// to optionally use local port forwarding.
```

Comment:
to use local port forwarding if requested.

?

Reply:
sure